### PR TITLE
Feat/create button component

### PR DESCRIPTION
**Pull request by IagoWag**
# 🚨 Fill in the fields below

[Add Ticket link](https://example.com)

## Description

```
~> Added Button component
```

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] This is an example checked

- [x] PRODUCT

  - [x] Put PR link on ticket
  - [x] Move the ticket

- [x] DEVELOPERS

  - [x] My code follows the style guidelines of this project
  - [x] I have performed a self-review of my own code
  

- [x] REVIEW
  - [x] Add reviewers: The entire team and most influential people in the area/project
  - [x] Assignees myself to it

🚨 - `Approve: Wait at least one influential in the project or more experienced`

# FRONTEND

### CODE REVIEW

File: src/components/Button/types.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import {TouchableOpacityProps} from 'react-native';
+
+export interface ButtonProps extends TouchableOpacityProps {
+  onPress: () => void;
```
Comment (JeanSilvany) [inline src/components/Button/types.ts:4]: Acrescentar interface:

label: string

File: src/screens/Login/index.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {Container, HeaderImageLogo, HeaderLogo} from './styles';
+import headerLogo from '../../assets/images/headerLogo.gif';
```
Comment (JeanSilvany) [inline src/screens/Login/index.tsx:3]: Componente inicia como Letra maiúscula.

File: src/@types/images.d.ts [hunk 3]
```diff
@@ -1 +1,2 @@
 declare module '*.png';
+declare module '*.gif';
```
Comment (JeanSilvany) [inline src/@types/images.d.ts:2]: Não necessita dessa declaração, pode remover.

**Review by JeanSilvany — CHANGES_REQUESTED** [review]

File: src/components/Button/styles.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components/native';
+
+export const Container = styled.TouchableOpacity`
+  width: 250px;
+  height: 60px;
+  background-color: black;
+`;
+//temporario
```
Comment (IbraDajani) [inline src/components/Button/styles.ts:8]: If you need to add a comment, use a comment like `//TODO temporario` or a `//FIXME temporario`
```suggestion
```

**Pull request merged**